core/modules/cart/features/cartPayment.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Functionality for returning payment method
+ *
+ * #### Computed properties
+ * - **`cartPayment`** returns cart payment. Returns `cart/payment` Vuex state.
```

Comment:
It's not very descriptive. Maybe we should specify that it returns default cart payment method?